rest-e2e-tests/Makefile

```diff
@@ -0,0 +1,78 @@
+#
+# https://makefiletutorial.com
+# https://clarkgrubb.com/makefile-style-guide
+#
+
+# This Makefile uses .ONESHELL option supported by Make 3.82
+ifeq ($(filter oneshell,$(.FEATURES)),)
+$(error error: Your version of make ($(shell make -v|head -1|cut -f 3 -d ' ')) does not support .ONESHELL)
+endif
+
+LANG := en_US.UTF-8
+SHELL := bash
+.SHELLFLAGS := -eu -o pipefail -c # run '/bin/bash ... -c /bin/cmd'
+.ONESHELL:
+.DEFAULT_GOAL := all
+
+nvm_brew = /usr/local/opt/nvm/nvm.sh
+ifneq ("$(wildcard $(nvm_brew))", "")
+	nvm_sh = $(nvm_brew)
+endif
+nvm_default = $(HOME)/.nvm/nvm.sh
+ifneq ("$(wildcard $(nvm_default))", "")
+	nvm_sh = $(nvm_default)
+endif
+node_version = $(shell cat .nvmrc)
+define npm
+	@$(eval npm_args=$(1))
+	bash -l -c "source $(nvm_sh) && nvm exec $(node_version) npm $(npm_args)"
+endef
+
+all: clean vendor test/server/wait test ## Set up and test
+
+node_modules: ## Run 'npm ci' if directory doesn't exist
+	$(call npm, ci)
+
+vendor: node_modules ## Install source code packages
+
+engine_status_check_url := http://localhost:8081/streamr-core/api/v1/users/me
```

Comment:
Use http://localhost/api/v1/users/me

Reply:
Changing url breaks the logic of the script. Nginx replies too quick while engine is down.